assets/scss/dark/_variables.scss

```diff
@@ -0,0 +1,51 @@
+// --------------------------------------------------
+// Sidebar
+// --------------------------------------------------
+$td-sidebar-section-title-color: $gray-400;
+$td-sidebar-link-color: $gray-400;
+$td-sidebar-bg-color: rgba($primary, 0.03);
+$td-sidebar-border-color: $border-color;
+$td-sidebar-closed-color: $gray-500;
+$td-sidebar-left-stroke-color: $gray-600;
+$td-sidebar-left-stroke-active-color: transparentize($primary, 0.8);
+
+// --------------------------------------------------
+// Search
+// --------------------------------------------------
+$search-icon-color: $teal;
+$search-placeholder-color: darken($teal, 10%);
+$search-border-color: darken($teal, 20%);
```

Comment:
For me the search bar right now green even in light mode:

<img width="772" alt="Screenshot 2025-03-11 at 19 42 56" src="https://github.com/user-attachments/assets/cab0f61a-2c4c-451a-9e7c-ab3565067560" />

Reply:
The idea was to highlight it because many users miss the fact we even have a search bar.

I will defer to @ederign if he thinks it looks better/worse.

Reply:
@thesuperzapper I understand that the search is kind of hidden, but I would not use a green color here.

For the dark theme, I believe if we put the border, icon, and text in white, it would have a similar effect. We can also increase the border for 2px to highlight it.

Reply:
I have updated the search styles in https://github.com/kubeflow/website/pull/3981/commits/e2d8a1cb9c6ab5588cbab857ae99e561e8eaaf9c so the green border is only visible when the search input is in focus.

I used a drop-shadow to highlight the presence of the search bar, so people are less likely to miss it.

> <img width="265" alt="Screenshot 2025-03-29 at 17 02 34" src="https://github.com/user-attachments/assets/d86c36e3-951b-4386-8e80-cdc1e9f6d25d" />

> <img width="271" alt="Screenshot 2025-03-29 at 17 02 28" src="https://github.com/user-attachments/assets/0a9d79aa-7be7-422c-a56a-ed43c4cf0f8a" />